benchmark/type/partition/brigand.list.cpp.erb

```diff
@@ -0,0 +1,21 @@
+// Copyright Odin Holmes 2016
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE.md or copy at http://boost.org/LICENSE_1_0.txt)
+
+#include <brigand/algorithm/partition.hpp>
+#include <brigand/sequences/list.hpp>
+#include <brigand/types/integer.hpp>
+#include <brigand/types/bool.hpp>
```

Comment:
out of curiosity, is there a include-it-all header for brigand?
you might consider including everything instead of selecting the specific header files for each benchmark, since metabench subtracts the time taken to preprocess header files and including everything won't affect results